launcher/ui/pages/global/LauncherPage.cpp

```diff
@@ -370,7 +375,7 @@ void LauncherPage::loadSettings()
     if (BuildConfig.UPDATER_ENABLED && APPLICATION->updateChecker()->getExternalUpdater())
     {
         ui->autoUpdateCheckBox->setChecked(
-                APPLICATION->updateChecker()->getExternalUpdater()->getAutomaticallyChecksForUpdates());
+                    APPLICATION->updateChecker()->getExternalUpdater()->getAutomaticallyChecksForUpdates());
```

Comment:
What's happened here

Reply:
just pressed enter, supposed it would go back to how it was. Doubt this really affects the pr

Reply:
I don't really want the pr to be modifying some random formatting in a completely unrelated part of the code in the same commit that it's adding specific functionality, it's not good.